src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -1349,22 +1349,29 @@ private bool TryGetMentorNode(string dbName, DatabaseTopology topology, ClusterT
                 return (false, null);
             }
 
-            var mentorsEtag = mentorPrevDbStats.LastEtag;
-            if (mentorCurrDbStats.LastSentEtag.TryGetValue(promotable, out var lastSentEtag) == false)
+            var mentorLastDbEtag = mentorPrevDbStats.LastEtag;
+            if (mentorCurrDbStats.LastSentEtag.TryGetValue(promotable, out var mentorLastDocsEtag) == false)
             {
-                LogMessage($"Can't find last sent etag of mentor {mentorNode} for {promotable}", database: dbName);
+                LogMessage($"Can't find last sent documents etag of mentor {mentorNode} for {promotable}", database: dbName);
+                return (false, null);
+            }
+
+            var promotableLastDbEtag = mentorPrevDbStats.LastEtag;
```

Comment:
how is that `promotableLastDbEtag` this is identical to `mentorLastDbEtag`

Reply:
You're right, I changed it to 'promotablePrevDbStats' and 'promotablePrevDbStats'

Reply:
I want to restore it to a specific node.